.travis.yml

```diff
@@ -6,12 +6,17 @@ go:
   - 1.7
 
 install:
+  - export BUILD_READY=false
```

Comment:
Would it make sense to use Travis [`env:`](https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml) to define the BUILD_READY environment variable?